### PR TITLE
[MIG][11.0] donation_bank_statement

### PR DESCRIPTION
**Pull request by tbaden**
ToDo: write Tests

### CODE REVIEW

File: donation_bank_statement/models/bank_statement.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                            vals = self._prepare_donation_vals(stline, mline)
+                            donation = ddo.create(vals)
+                            donation.partner_id_change()
+                            donation.line_ids.product_id_change()
+                            donation_ids.append(donation.id)
+
+    @api.multi
+    def button_confirm_bank(self):
+        statements = self.filtered(lambda st: st.state == 'open')
+        for stmt in statements:
```
Comment (umiphos) [inline donation_bank_statement/models/bank_statement.py]:
```suggestion
        for stmt in self.filtered(lambda st: st.state == 'open'):
```

There is no need for the variable

Comment (tbaden) [reply donation_bank_statement/models/bank_statement.py]: Thanks, Done

**Review by benbrich — APPROVED** [review]
I have tested the workflow from the beginning to the end and everything runs smooth as in v10.

**Comment by tbaden** [comment]
@umiphos some more changes or is everything ok?

**Comment by tbaden** [comment]
@umiphos thanks for the 👍 , an approve would be better to change the status of the PR :)

**Comment by tbaden** [comment]
added commit history to PR

**Comment by tbaden** [comment]
@pedrobaeza if it's ok with you, I'll improve the test coverage in an additional PR later on.

**Comment by pedrobaeza** [comment]
It's correct to let coverage as is for now, as it's not a requirement in migration.

**Comment by benbrich** [comment]
@umiphos can you please give your approve on this one, so it can be merged? Thx!

**Comment by umiphos** [comment]
@benbrich I can't give the approve, I'm just a random joe doing reviews on PRs(not a member)

**Comment by benbrich** [comment]
@umiphos AFAIK everyone can review and give an approve. Can you pls check? ...by clicking on the bubble on the right side of your name in the reviewers list (top of PR right side). On the top right side of the newly opened page you can find a green 'Review Changes' Button. After clicking you should be able to select the approve option in the selection field and confirm. Or am I wrong @pedrobaeza ?

**Review by umiphos — APPROVED** [review]

**Comment by tbaden** [comment]
@pedrobaeza this pr looks ready to merge. can you please check it.

**Comment by OSevangelist** [comment]
@pedrobaeza @alexis-via could anybody of you please merge this ? It seems to have been successfully discussed and approved

**Pull request closed without merging**